lib/internal/Magento/Framework/Interception/Config/Config.php

```diff
@@ -74,46 +69,46 @@ class Config implements \Magento\Framework\Interception\ConfigInterface
     protected $_scopeList;
 
     /**
-     * @var SerializerInterface
+     * @var CacheManager
      */
-    private $serializer;
+    private $cacheManager;
 
     /**
      * Config constructor
      *
      * @param \Magento\Framework\Config\ReaderInterface $reader
      * @param \Magento\Framework\Config\ScopeListInterface $scopeList
-     * @param \Magento\Framework\Cache\FrontendInterface $cache
+     * @param null $cache
      * @param \Magento\Framework\ObjectManager\RelationsInterface $relations
      * @param \Magento\Framework\Interception\ObjectManager\ConfigInterface $omConfig
      * @param \Magento\Framework\ObjectManager\DefinitionInterface $classDefinitions
      * @param string $cacheId
-     * @param SerializerInterface|null $serializer
+     * @param null $serializer
+     * @param CacheManager $cacheManager
      */
     public function __construct(
         \Magento\Framework\Config\ReaderInterface $reader,
         \Magento\Framework\Config\ScopeListInterface $scopeList,
-        \Magento\Framework\Cache\FrontendInterface $cache,
+        $cache = null,
```

Comment:
No, please dont change the signature of the constructor even if it is not used anymore

Reply:
Generally you cannot add optional argument between required ones.

Reply:
And I am sure that arguments should be marked as deprecated in order to remove the later. I cannot help you just now but you could search on Devdocs. Please ask me if you can't find that